en/deploy-tidb-binlog.md

```diff
@@ -396,6 +406,11 @@ The steps are as follows:
 
 ### Remove Pump nodes completely
 
+> **Note:**
+>
+> - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump cluster has been scaled in to `0` node at this time, you need to scale out the cluster at least to `1` node before you perform the removing operation in this section.
```

Comment:
I think "Pump cluster" is a bit strange, maybe we can just call it "Pump" and refer to the scale-in as "scale the Pump nodes to zero"?